internal/commands/exec/exec_test.go

```diff
@@ -32,22 +49,21 @@ func TestRunExecLoginError(t *testing.T) {
 		loginFail: true,
 	})
 	require.Equal(t, 1, RunExec(ExecConfig{
-		Context:      context.Background(),
-		Logger:       logger,
-		ConsulClient: consul.client,
-		ConsulConfig: *consul.config,
+		Context:            context.Background(),
+		Logger:             logger,
+		ConsulClient:       consul.client,
+		ConsulConfig:       *consul.config,
+		ConsulClientConfig: consul.clientConfig,
 		AuthConfig: AuthConfig{
 			Method: "nonexistent",
 			Token:  "token",
 		},
 		isTest: true,
 	}))
-	require.Contains(t, buffer.String(), "error logging into consul")
+	require.Contains(t, buffer.String(), "no such file or directory")
 }
 
 func TestRunExecLoginSuccessRegistrationFail(t *testing.T) {
-	t.Parallel()
```

Comment:
I'm assuming this was needed because of the un-threadsafe behavior of how consul-server-connection-manager is registering the gRPC load balancer? Refs https://github.com/hashicorp/consul-server-connection-manager/pull/25

Reply:
👍